resources/views/deletefaq.blade.php

```diff
@@ -0,0 +1,25 @@
+<?php
+use Illuminate\Support\Facades\DB
+?>
+
+<x-layout>
+    <x-slot:title>
+        Delete FAQ Post {{ $post }}
+    </x-slot>
+    <form method="post" action="/deletefaqpost">
+        @method('put')
+        @csrf
+        <h1>Vraag: </h1>
+        <input type="text" name="question" value="{{ DB::table('faq')->where('id', $post)->value('question') }}">
+        <h1>Antwoord: </h1>
+        <input type="text" name="answer" value="{{ DB::table('faq')->where('id', $post)->value('answer') }}">
```

Comment:
Calling `DB::table(...` twice in one view means querying the database twice for basically the same data. This makes your app inefficient. This can be solved easily by start using models instead. That is why Laravel added them